dotcom-rendering/src/web/server/pageTemplate.ts

```diff
@@ -296,6 +296,25 @@ https://workforus.theguardian.com/careers/product-engineering/
 					window.curl = window.curlConfig;
 				</script>
 
+				<script>
+					/** synthetically bork FID https://web.dev/fid/ */
+					if (window.location.hash.startsWith("#fid-borked-")) {
+					const delay = Number(window.location.hash.replace("#fid-borked-", ""));
+					const controller = new AbortController();
+					const options = { once: true, signal: controller.signal };
+					const bork = () => {
+						const now = performance.now();
+						while (performance.now() - now < delay) {
+						// throttling
+						}
+						controller.abort();
+					};
+					window.addEventListener("pointerdown", bork, options);
+					window.addEventListener("keydown", bork, options);
+					window.addEventListener("click", bork, options);
+					}
```

Comment:
maybe wrap this in a `try`/`catch` - don't want to break working pages

```suggestion
					/** synthetically bork FID https://web.dev/fid/ */
					if (window.location.hash.startsWith("#fid-borked-")) {
						try {
							const delay = Number(window.location.hash.replace("#fid-borked-", ""));
							const controller = new AbortController();
							const options = { once: true, signal: controller.signal };
							const bork = () => {
								const now = performance.now();
								while (performance.now() - now < delay) {
								// throttling
								}
								controller.abort();
							};
							window.addEventListener("pointerdown", bork, options);
							window.addEventListener("keydown", bork, options);
							window.addEventListener("click", bork, options);
						} catch (e) {
							// do nothing
						}
					}
```